src/assets/wise5/authoringTool/node/editRubric/edit-rubric.component.ts

```diff
@@ -1,50 +1,42 @@
+import { Component, OnInit } from '@angular/core';
 import { insertWiseLinks, replaceWiseLinks } from '../../../common/wise-link/wise-link';
 import { ConfigService } from '../../../services/configService';
 import { TeacherDataService } from '../../../services/teacherDataService';
 import { TeacherProjectService } from '../../../services/teacherProjectService';
+import { UpgradeModule } from '@angular/upgrade/static';
 
-class EditRubricComponentController {
+@Component({
+  selector: 'edit-rubric',
+  templateUrl: 'edit-rubric.component.html'
+})
+export class EditRubricComponent implements OnInit {
```

Comment:
Should this class be named EditNodeRubricComponent to distinguish from EditComponentRubricComponent?